packages/ragbits-core/src/ragbits/core/vector_stores/chroma.py

```diff
@@ -32,24 +44,29 @@ def __init__(
             index_name: The name of the index.
             distance_method: The distance method to use.
             default_options: The default options for querying the vector store.
+            metadata_store: The metadata store to use.
         """
-        super().__init__(default_options)
+        super().__init__(default_options, metadata_store)
         self._client = client
         self._index_name = index_name
         self._distance_method = distance_method
-        self._collection = self._get_chroma_collection()
+        self._collection: Collection | None = None
```

Comment:
Why change here?

Reply:
When I had global metastore with async methods I needed to do a lazy loading, should have deleted it with previous commit, thanks for pointing it out.